package.json

```diff
@@ -82,16 +82,18 @@
   ],
   "jspm": {
     "main": "js/bootstrap",
+    "ignore": ["dist/js/npm"],
+    "directories": {
+      "lib": "dist"
```

Comment:
I can't think of a solid reason people would want the raw ES6 files- did you have a use case in mind?

People will want the ability to load the individual bootstrap components- which dist/js/umd/... enables.